### ISSUE TITLE
Cheats still not working

### ISSUE THREAD

**Issue by efrainm23** [libretro/beetle-saturn-libretro#5]
Its good to see that this core has been updated to latest 1.29.0 version. However, cheats had not been working with the previous core version and they still do not with this new update. The same codes work fine on stand alone Mednafen. There was an open issue for this problem with the previous version before this rebase.  

I have been using Keio Flying Squadron 2 (Europe) as a test using the following cheats:

cheat0_desc = "9999 Bunnies Collected"
cheat0_code = "1020004E 270F"
cheat0_enable = true

cheat1_desc = "99 Lives"
cheat1_code = "30200013 0063"
cheat1_enable = true

Just for comparison, I believe the Kronos core based on Yabause does have working cheats last I checked. The one I am certain does have working cheats is the Yaba Sanshiro core also based on Yabause.  Those cores are obviously less accurate and have more broken games. Its a shame, the non working cheats is the last thing keeping me from ditching stand alone Mednafen and converting my collection for Saturn to CHD.

**Comment by LIEV-008** [comment]
I completely agree.  Cheats support is a necessary part for any core.  I don't understand why you are taking so long with this.

**Comment by sonninnos** [comment]
Just tried searching and creating my own cheats, and that worked fine.

**Comment by efrainm23** [comment]
I tried the script referenced in https://github.com/libretro/libretro-database/issues/1360 , and it does work to convert the cheats to a format that Beetle Saturn will work with. Some of the cheats in the Retroarch cheat database still gave me issues with the conversion script, but ones I looked up from gamehackng.org and downloaded in .cht format did work fine. 

I will just have to convert them on a game by game basis as needed. Just like in the link referenced, maybe someday Retroarch can handle these kinds of cores with these cheat issues better in the future with the native cheat codes converted the the Retroarch format without needing to be converted.

**Comment by LibretroAdmin** [comment]
Finally taking a stab at this
https://github.com/libretro/beetle-saturn-libretro/commit/334be520e62c375814157f87715bf5ffbd75960b#diff-d2337877636464e2576d496bd6e932c6a20fc4c3dded80cefcfc91c0de0dc07a
